fem/bilininteg_dgtrace_pa.cpp

```diff
@@ -182,22 +182,22 @@ void DGTraceIntegrator::SetupPA(const FiniteElementSpace &fes, FaceType type)
       auto C = Reshape(vel.HostWrite(), dim, nq, nf);
       Vector Vq(dim);
       int f_ind = 0;
-      for (int f = 0; f < fes.GetNF(); ++f)
+      for (int f = 0; f < mesh->GetNumFacesWithGhost(); ++f)
       {
-         int e1, e2;
-         int inf1, inf2;
-         fes.GetMesh()->GetFaceElements(f, &e1, &e2);
-         fes.GetMesh()->GetFaceInfos(f, &inf1, &inf2);
-         int face_id = inf1 / 64;
-         if ((type==FaceType::Interior && (e2>=0 || (e2<0 && inf2>=0))) ||
-             (type==FaceType::Boundary && e2<0 && inf2<0) )
+         Mesh::FaceInformation face = mesh->GetFaceInformation(f);
+         if (face.IsLocal() && face.IsNonConformingMaster())
+         {
+            // We skip local non-conforming master faces
+            continue;
+         }
+         else if ( face.IsOfFaceType(type) )
          {
             FaceElementTransformations &T =
-               *fes.GetMesh()->GetFaceElementTransformations(f);
+               *fes.GetMesh()->GetFaceElementTransformations(f,5); // Elem1 and Loc1
```

Comment:
Can 5 use a symbol to be more self-describing? Maybe from an enum?

Reply:
https://github.com/mfem/mfem/pull/2195#discussion_r745072458

Reply:
I used a `const int mask` variable to make the code more self documented.